test/index.test.js

```diff
@@ -0,0 +1,34 @@
+jest.mock('../ffc-pay-file-sender/config', () => ({ totalRetries: 1 }))
+const sender = require('../ffc-pay-file-sender')
+const mockContext = require('./mock-context')
+jest.mock('../ffc-pay-file-sender/storage')
+const mockStorage = require('../ffc-pay-file-sender/storage')
+let message
+const blob = {}
+const content = 'content'
+
+describe('sender', () => {
```

Comment:
jest.mock('../ffc-pay-file-sender/config', () => ({ totalRetries: 1 }))
jest.mock('../ffc-pay-file-sender/storage')
const mockStorage = require('../ffc-pay-file-sender/storage')
const mockContext = require('./mock-context')

const sender = require('../ffc-pay-file-sender')

const blob = {}
const content = 'content' // suggest a comment to define that this is merely the console/I assume whatever runtime env its in?

let message

describe('sender', () => {

Reply:
Not sure I understand this?  It represents the content of the blob file rather than anything runtime related

Reply:
Aaaahhhh, ok, ok, my mistake. I thought it was referring to `context`, i.e. `context.log('...')` in `index.js` which refers to the Azure Function runtime.

Got my conteNT mixed up with my conteXT ahahaha

Reply:
A suggestion for structuring the mocks and imports

`jest.mock('../ffc-pay-file-sender/config', () => ({ totalRetries: 1 }))`
`jest.mock('../ffc-pay-file-sender/storage')`
`const mockStorage = require('../ffc-pay-file-sender/storage')`
`const mockContext = require('./mock-context')`

`const sender = require('../ffc-pay-file-sender')`

`const blob = {}`
`const content = 'content'`

`let message`

`describe('sender', () => {`

Reply:
I've changed this to match 99% of what you have above.  The only thing I've done is keep the mock and their requires together.  Agree your suggestion is better for readibility, but depending on how you're using them, Jest mocks don't always work if they're not immediately before the require line